src/ext/ddtrace.c

```diff
@@ -545,6 +548,13 @@ static PHP_FUNCTION(dd_trace) {
         DD_PRINTF("Class name: %s", Z_STRVAL_P(class_name));
     }
     DD_PRINTF("Function name: %s", Z_STRVAL_P(function));
+    if (ddtrace_blacklisted_disable_legacy && !DDTRACE_G(ignore_legacy_blacklist)) {
+        ddtrace_log_debugf(
+            "Cannot instrument '%s()' with dd_trace(). This functionality is disabled due to a potentially conflicting "
+            "module. To re-enable dd_trace(), please set the INI setting: ddtrace.ignore_legacy_blacklist=1",
+            Z_STRVAL_P(function));
```

Comment:
We discussed limiting this error to once per request; what do you think?

Reply:
We can totally do that. But I'm leaning towards emitting the message for every call so that it's easier to find all the places that need upgrading to the sandbox API.

Reply:
Of course I'm open to emitting a generic message once per request as well. :)